src/button-groups/button-radio.ts

```diff
@@ -0,0 +1,41 @@
+import { Directive, Input, HostListener, ElementRef, Renderer } from 'angular2/core';
+import { Subscription } from 'rxjs/Subscription';
+import { NglButtonGroup } from './button-group';
+
+@Directive({
+  selector: '[nglButtonRadio]',
+})
+export class NglButtonRadio {
+  value: any;
+
+  constructor(public element: ElementRef, public renderer: Renderer, public group: NglButtonGroup) { }
```

Comment:
Like above, I'd suggest turning `element`, `renderer` and `group` to private members for better encapsulation.